src/swarm/client/helper/NodesConfigReader.d

```diff
@@ -191,13 +191,13 @@ static:
         # 127.0.0.3:6000";
 
         auto res = read(content);
-        assert(res.length == 2);
+        test(res.length == 2);
```

Comment:
I think the intended usage is `test!("==")(res.length, 2)`.

Reply:
Yes, I've missed all these.